index.bs

```diff
@@ -373,7 +373,25 @@ contexts. This allows developers to detect support for the Layout Instability AP
   };
 </pre>
 
-All attributes have the values which are assigned to them by the steps to
+Each {{LayoutShiftAttribution}} is associated with a {{Node}} (its
+<dfn>associated node</dfn>).
+
+The getter of the <a href="#dom-layoutshiftattribution-node">node</a> attribute
```

Comment:
Note that this means that the interface must be `readonly attribute Node? node` (nullable), as the return value can be null. Could you change the interface?

On unrelated note, I now just notice that this is called on a Node whereas the other algorithm assumes it's an Element, but I think that is a minor issue which we can fix once that method is integrated into the DOM spec.

Reply:
Done, and marked an issue to generalize the algorithm to Node.